decidim-proposals/app/views/decidim/proposals/admin/proposals/bulk_actions/_dropdown.html.erb

```diff
@@ -49,6 +49,11 @@
           </button>
         </li>
       <% end %>
+      <li>
+        <button type="button" data-action="change-answer-status">
+          <%= t("decidim.proposals.admin.proposals.index.change_answer_status") %>
+        </button>
+      </li>
```

Comment:
This option should only appear if there are any templates to be applied.
Also, let's use something like `apply-answer-template`instead of `change-answer-status`